src/main/webapp/WEB-INF/clinics-data.csv

```diff
@@ -1,10 +1,34 @@
-43.514517,-96.80785,Sioux Falls Clinic,6511 West 41st Street,(605) 361-5100,"Abortion Services ; Birth Control ; HIV Testing ; Men's Health Care ; Morning-After Pill (Emergency Contraception) ; Pregnancy Testing & Services ; STD Testing, Treatment & Vaccines ; Women's Health Care","Patients seeking emergency contraception (morning after pill) and birth control refill pick-ups may walk-in anytime the health center is open. You may also walk into the health center at least one hour before it closes for pregnancy tests and rapid HIV tests. Appointments You can privately and easily book your appointment online for select services 24/7. For the full set of services, you can call605-361-5100to book your appointment. Health center staff can also answer any questions you may have. Book Appointment Call 605-361-5100 Insurance & Payments Everyone deserves affordable health care. With or without insurance, you can always come to us for your health care. I don’t have insurance or Medicaid If you don't have insurance, you may qualify for a state-funded program or a lower fee scale. If you’re worried about cost, talk to us about how you can get health care that fits your budget. Get Coverage Our fees are based on your household income. I have insurance or Medicaid We take the following insurance: Aetna Blue Cross Blue Shield DakotaCare HealthPartners Medica Preferred One Sanford United Healthcare Wellmark Hennepin Health We accept many private and public insurance plans. If you don't see your insurance listed, please call us at 605-361-5100. Payment Options Most health insurance plans now cover prescription birth control, annual wellness exams, and HIV and STD screenings with no copay, and many other services with some copay required. Contact your health insurance company directly to confirm that the services you need are covered and what, if any, out of pocket costs you are required to pay. Additional Payment information Payment is expected at time of service. We accept: Cash Check Major Credit/Debit Cards Money Orders If you do not have health insurance, read about health insurance optionsto find out how to get more affordable coverage and what to consider when choosing a plan. clipboard Additional Information",2738
\ No newline at end of file
+47.51836784,-92.55314749,Virginia Clinic,505 12th Avenue West,(218) 741-8192,"Abortion Referrals ; Birth Control ; HIV Testing ; Men's Health Care ; Morning-After Pill (Emergency Contraception) ; Pregnancy Testing & Services ; STD Testing, Treatment & Vaccines ; Women's Health Care"

```

Comment:
Do you still need the clinics-data.csv? Do you store the data in two separate files?